setup.cfg

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python                                                           
+# -*- coding: utf-8 -*-                                                         
+#                                                                               
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.                          
+# SPDX-License-Identifier: BSD-2-Clause
+
+[metadata]
+name = tern
+author = VMWare Inc
```

Comment:
VMware is very particular about the small "w" after the big "M". This should be "VMware, Inc."
Sorry I didn't catch that before.